### DISCUSSION TITLE
Multiselect throws error when double clicking on selection container

### DISCUSSION THREAD

**Discussion by turp** [primefaces/community#173] [PrimeNG] [answered]
I setup a simple multiselect. If you quickly click a few times in the whitespace of the p-multiselect-label-container, Angular will eventually throw this error that shows up in the browser console:

`ERROR Error: NG0100: ExpressionChangedAfterItHasBeenCheckedError: Expression has changed after it was checked. Previous value: 'false'. Current value: 'true'. Expression location: MultiSelect component
`

Here is the code I used to duplicate the issue: https://stackblitz.com/edit/dtvhna?file=src%2Fapp%2Fdemo%2Fmulti-select-virtual-scroll-demo.ts

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Thank you for sharing an example and reporting the issue, I've just converted the topic into an issue on github you can track the issue from here https://github.com/primefaces/primeng/issues/13808
